### DISCUSSION TITLE
Is there an equivalent of the Cellar for casks?

### DISCUSSION THREAD

**Discussion by alexreg** [Homebrew/discussions#276] [Casks]
That is, does `brew install [cask]` ever write to somewhere analogous to what `/usr/local/Cellar` is for normal formulas? Or is it purely defined by `pkg` installer or a script in all cases?

**Comment by carlocab** [comment]
There is `Caskroom`, but it's not exactly like `Cellar`, since the `cask` apps you run are not symlinks out of the `Caskroom`.

I believe `brew cask` used to use symlinks out of this much like how `brew` used `Cellar`, but that broke too many casks so they changed it. Not to mention that some package installers just can't be told where you want them to install things.

**Reply by alexreg** [reply]
Ah, I see, thanks. It seems like it might just be a staging area of some sort now, if I'm not mistaken? The problem I'm trying to get around is `brewdo` running the installer as `_homebrew` user, which is normally fine, but likes to mess with the local user for a few things, occasionally. Rather annoying.

**Reply by gromgit** [reply]
AIUI, `/usr/local/Caskroom` is, at minimum, used to track the installed version of each cask, and contain whatever is left over from the installer medium. Since it's created at Homebrew install time, there's no reason why it would have a different owner from the rest of Homebrew.

**Reply by alexreg** [reply]
Okay, that's good to know, thanks.
